### PR TITLE
fix yaml indentation 

### PR DESCRIPTION
**Pull request by mirnawong1**
update semantic mode yaml code

https://github.com/dbt-labs/jaffle-sl-template/blob/main/models/metrics/example_metrics.yaml

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @Jstein77 , in the [dimensions](https://docs.getdbt.com/docs/build/dimensions#time) page, should we remove the `is_primary` tab and code?

File: website/docs/docs/build/derived-metrics.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ Derived metrics in MetricFlow refer to metrics that are created by defining an e
...
-        alias: active_users # Optional alias to use in the expr
+  - name: net_sales_per_user
+    type: derived
+    type_params:
+      expr: gross_sales - cogs / active_users
+      metrics:
+        - name: gross_sales # these are all metrics (can be a derived metric, meaning building a derived metric with derived metrics)
+        - name: cogs
+        - name: users
+          filter: is_active # Optional additional constraint
```
Comment (Jstein77) [inline website/docs/docs/build/derived-metrics.md]: Nit:  {{dimension('filter')}}

File: website/docs/docs/build/metrics-overview.md [hunk 2]
```diff
@@ -124,13 +124,13 @@ metrics:
...
+  - name: cancellations 
+    type: simple 
+    type_params:
   # Specify the measure you are creating a proxy for. 
-    measure: cancellations_usd 
-  filter: | 
-    value > 100 AND user__acquisition
+      measure: cancellations_usd 
+      filter: | 
+      value > 100 AND user__acquisition
```
Comment (Jstein77) [inline website/docs/docs/build/metrics-overview.md]: filter: |
 {{dimension('value')}} > 100 and {{dimension('acquisition', entity_path=['user'])}}

File: website/docs/docs/build/simple.md [hunk 3]
```diff
@@ -16,11 +16,11 @@ If you've already defined the measure using the `create_metric: true` parameter,
...
-    measure: cancellations_usd # The measure you're creating a proxy of.
+  - name: cancellations
+    type: simple # Pointers to a measure you created in a data source
+    type_params:
+      measure: cancellations_usd # The measure you're creating a proxy of.
   # For any metric optionally include a filter string which applies a dimensional filter when computing the metric
-  filter: | 
-    value > 100 and user__acquisition
+      filter: | 
+      value > 100 and user__acquisition
```
Comment (Jstein77) [inline website/docs/docs/build/simple.md]: {{dimension('value')}} > 100 and {{dimension('acquisition', entity_path=['user'])}}

**Review by Jstein77 — APPROVED** [review]
Approved with a few requested changes.

File: website/docs/docs/build/metrics-overview.md (hunk 2 above)
Comment (mirnawong1) [inline website/docs/docs/build/metrics-overview.md]:
```suggestion
      {{dimension('value')}} > 100 and {{dimension('acquisition', entity_path=['user'])}}
```

File: website/docs/docs/build/simple.md (hunk 3 above)
Comment (mirnawong1) [inline website/docs/docs/build/simple.md]:
```suggestion
      {{dimension('value')}} > 100 and {{dimension('acquisition', entity_path=['user'])}}
```

File: website/docs/docs/build/derived-metrics.md (hunk 1 above)
Comment (mirnawong1) [inline website/docs/docs/build/derived-metrics.md]:
```suggestion
          filter: | # Optional additional constraint
          {{dimension('filter')}} is_active
```

**Comment by mirnawong1** [comment]
thanks so much @Jstein77 , made those changes ✨merging now!

**Pull request merged**